src/devtools/client/debugger/src/components/Editor/EditorPane.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import classNames from "classnames";
+import { useSelector } from "react-redux";
+import Editor from "./index";
+import EditorTabs from "./Tabs";
+import EditorFooter from "./Footer";
+
+import WelcomeBox from "../WelcomeBox";
+import { getSelectedSource, getOrientation } from "../../selectors";
+
+import { Redacted } from "ui/components/Redacted";
+import useWidthObserver from "ui/utils/useWidthObserver";
+
+const Pane = ({ children }) => {
+  const [paneNode, setPanelNode] = useState(null);
+  const nodeWidth = useWidthObserver(paneNode);
+
+  return (
+    <div
+      className={classNames("editor-pane overflow-hidden rounded-lg", {
+        narrow: nodeWidth && nodeWidth < 240,
+      })}
+      ref={node => setPanelNode(node)}
```

Comment:
A place for a ref, not a state (`useRef`).

Reply:
good call. Updated